### PR TITLE
npm dev findings

### PR DESCRIPTION
**Pull request by sumanthkb44**
Additional logic for finding Dev Components in NPM & syft installation with specific version

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -13,13 +13,14 @@ RUN mkdir /opt/DebianImageClearing && \
...
 
 # Installing required packages
+# Installing syft:v0.90.0
 RUN apt-get update && \
     apt-get -y install --no-install-recommends nodejs npm && \
     apt-get -y install --no-install-recommends git && \
     apt-get -y install --no-install-recommends maven && \
     apt-get -y install --no-install-recommends curl && \
     apt-get -y install --no-install-recommends dpkg-dev && \   
-    curl -sSfL https://raw.githubusercontent.com/anchore/syft/main/install.sh | sh -s -- -b /opt/DebianImageClearing && \
```
Comment (karthika-g) [inline Dockerfile:22]: is this change required?

Comment (sumanthkb44) [reply Dockerfile:22]: Required

**Review by karthika-g — APPROVED** [review]
approved
